Update documentation regarding Helm Chart v3 - part 1 (#1659)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/kubernetes-control-plane.md

All the dashboards are linked to the Explore tab and can be easily accessed by clicking the Cluster, in the left navigation pane of Explore tab.

### Step 1: Determine custom data filters for source categories

The [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection) uses the [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) to collect and enrich logs.
Based on the configuration some information are extracted as metadata added to logs, one of the special metadata is `Source Category` which helps to search data.

To determine custom data filters for source categories, do the following:

> 1. Run a query similar to the following in Sumo Logic to determine the source categories. Use the Collector Name which you configured for Kubernetes collection.

Reviewer: ```suggestion
1. Run a query similar to the following in Sumo Logic to determine the source categories. Use the Collector Name that you configured for Kubernetes collection.
```

Author: changed

All the dashboards are linked to the Explore tab and can be easily accessed by clicking the Cluster, in the left navigation pane of Explore tab.

### Step 1: Determine custom data filters for source categories

When you installed the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection) in your cluster,
logs are collected by [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).

> Based on the configuration some information are extracted as metadata added to logs, one of the special metadata is `Source Category` which helps to search data.

Reviewer 2: ```suggestion
We add several built-in metadata fields to all ingested log records. We're going to use the `Source Category` field for querying in the following steps.
```

Author: This proposition does not match the style. It will be the only sentence with "we.

Author: @kimsauce Could you help here?

Reviewer: I agree @kkujawa-sumo . Using "we" would make it inconsistent with the voice of this particular doc.

Reviewer 2: How about
```suggestion
All ingested log records will have certain built-in metadata fields set. One of them is the `Source Category` field, which can be used to specify custom data filters for this app.
```

Author: Changed to @swiatekm-sumo's  proposition.
